services/visual_recognition/v3.js

```diff
@@ -258,14 +273,24 @@ module.exports = function (RED) {
             return;        
           }  
           stream_buffer(info.path, msg.payload, function () {
-          params['images_file'] = fs.createReadStream(info.path);
-          if ( msg.params != null && msg.params.classifier_ids != null)
-            params['classifier_ids']=msg.params['classifier_ids'];
-          if ( msg.params != null && msg.params.owners != null)
-            params['owners']=msg.params['owners'];
-          if ( msg.params != null && msg.params.threshold != null)
-            params['threshold']=msg.params['threshold'];
-          performAction(params, feature, actionComplete);
+            params['images_file'] = fs.createReadStream(info.path);
+            if ( msg.params != null && msg.params.classifier_ids != null)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'classifier_ids' is not in camel case.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2714380600/issues/source?bid=3330694&fileBranchId=3344618#l277)